observablehq.config.ts

```diff
@@ -10,6 +10,7 @@ export default {
     {name: "SQL", path: "/sql"},
     {name: "Themes", path: "/themes"},
     {name: "Configuration", path: "/config"},
+    {name: "Deploying", path: "/deploying"},
```

Comment:
Does this need to be listed in the sidebar? I would think not (like for /search, it's not something you have to keep going back to once it's set up).

Reply:
I think if we don't link it in the sidebar, people aren't likely to find it as much as we want them to. Considering this is the only page where we explain several key points of automatic deploys (like authentication and caching) I think having it discoverable in the navigation is important.

Reply:
I'm not fully convinced but it's not a blocker :)